docs/tools/fbsql/fbsql-loaders.md

```diff
@@ -0,0 +1,53 @@
+---
+title: fbsql-loaders
+layout: default
+parent: fbsql
+grand_parent: Tools
+nav_order: 2
+---
+
+# FBSQL Loaders
+
+`fbsql` is a CLI tool used to interact with FeatureBase using SQL. It can also be run in non-interactive mode to load data from supported data sources. These "loaders" connect to a data source, build batches of SQL `BULK INSERT` statements with the data returned from the source, and send those `BULK INSERT` requests to FeatureBase.
```

Comment:
### issues

Code block for "fbsql" seems unnecessary because it's not code, it's a tool. Just set it to bold.

"interact with FeatureBase using SQL" -- would it be safe to say the user is running SQL queries and statements? It could be simpler to say that.

"it can also be run in non-interactive mode" would be better left off because there's *always* an interaction, even if the user is specifying the data source using the loaders.

Don't double-quote **loaders** because that implies it's a euphemism and **not** the specific thing. 

Also, this sentence is restating what's already said -- that the loaders connect to external data sources.

Single word hyperlink anchor text is bad (full explaination in help-on-help: https://github.com/FeatureBaseDB/featurebase-docs/blob/main/help-on-help/writing-help/writing-hyperlinks.md )

### here's a suggested rewrite

It appears you're saying the following:

```
**Fbsql** is a FeatureBase CLI tool used to run statements and queries using supported SQL.

You can also use **fbsql** to load data from supported data sources including the following:

* [Kafka fbsql data source loader](/docs/tools/fbsql/fbsql-loaders-kafka)
* [PostgreSQL data source loader](/docs/tools/fbsql/fbsql-loaders-postgresql)
* [Impala data source loader](/docs/tools/fbsql/fbsql-loaders-impala)
```

I'm leaving off the `--loader[data-source]` information because that's restated in the destination pages and there's no point telling the user to go somewhere else if you're explaining some of it here.